//package core.io.protocol;

message DseStageReset
{
	optional int32		res = 1;//结果
}

